packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -488,9 +488,9 @@ class _BottomNavigationTile extends StatelessWidget {
       ),
     );
 
-    if (item.label != null) {
+    if (item.toolTip != null) {
```

Comment:
Since `toolTip` is a new parameter and defaults to null, this breaks bottom navigation bars that need to show tips.

Reply:
See https://github.com/flutter/flutter/pull/71079#issuecomment-734463362 for a potential workaround.

Reply:
Setting it to an empty string results into something like this : 
![Screenshot_2020-12-11_22-37-08](https://user-images.githubusercontent.com/38391570/101933245-ed5f1f00-3c01-11eb-9096-8fc25b8f30da.png)

Reply:
Maybe that's a bug in Tooltip, it mustn't show tip with an empty string.

Reply:
I think the suggestion was to check if the tooltip string is empty, if it is don't include the tooltip. Ideally we want the default value to represent the "inherit the tooltip string from `label`" option so it doesn't break existing codebases that use `BottomNavigationBarItem`, and pick another special value that represents "don't show the tooltip".

Reply:
I have added a test which states
* if tooltip = '', then tooltip doesn't show up.
* if tooltip = null or not specified, then it defaults to `label`.
* if tooltip = 'msg', then a Tooltip with `msg` will be shown.